explorer/idl/explorer.idl

```diff
@@ -38,10 +38,8 @@ struct Block {
     executions int
     generateBy BlockGenerator
     amount string
```

Comment:
should deprecate api v1 after this?

Reply:
We can label it as a breaking change first. Explorer on master branch is already broken before this PR. We can deprecate it now to avoid further changes on explorer backend.